package/yast2-theme.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Sun Jan  6 09:36:39 UTC 2019 - Noah Davis <noahadvs@gmail.com>
+
+- Add Breeze pattern icons
```

Comment:
Any bug number to reference? Our Jenkins script would reject this change otherwise....

Reply:
Added it to the title